packages/paste-website/src/component-examples/KeyboardKeyExamples.ts

```diff
@@ -0,0 +1,191 @@
+                    </KeyboardKeyGroup>
+                </Box>
+            </MenuItem>
+            <MenuItem disabled aria-disabled {...menu}>
+                <Box display="flex" justifyContent="space-between">
+                    Save
+                    <KeyboardKeyGroup disabled>
+                    <KeyboardKey>Cmd</KeyboardKey>
+                    <KeyboardKey>S</KeyboardKey>
+                    </KeyboardKeyGroup>
+                </Box>
+            </MenuItem>
+        </Menu>
+    </Box>
+  );
+};
+
+render(
+    <DisabledExample />
+)`.trim();
+
+export const pressedExample = `const PressedExample = () => {
+    const keyCombinationState = useKeyCombination({
+        keys: ["Shift", "s"],
+        onCombinationPress: ()=> {},
+        enablePressStyles: true,
+    });
+
+    return (
+        <>
+            <KeyboardKeyGroup enablePressStyles activeKeys={["s"]}>
+                <KeyboardKey keyEvent="s">S</KeyboardKey>
+            </KeyboardKeyGroup>
+            <Paragraph>Press the “Shift” or “S” key to reveal the pressed states below:</Paragraph>
+            <KeyboardKeyGroup {...keyCombinationState}>
+                <KeyboardKey keyEvent="Shift">Shift</KeyboardKey>
+                <KeyboardKey keyEvent="s">S</KeyboardKey>
+            </KeyboardKeyGroup>
+        </>
+    )
+}
+
+render(
+    <PressedExample />
+)`.trim();
+
+export const tooltipExample = `<Stack orientation="horizontal" spacing="space60">
+    <Tooltip
+        actionHeader="Search shortcuts"
+        keyCombinationsActions={[
+            { name: "Mac", eventKeyCombination: ["Command", "K"] },
+            { name: "Windows", eventKeyCombination: ["Control", "K"] },
+        ]}
+    >
+        <Button aria-keyshortcuts="command+k" variant="primary">
+            <SearchIcon title="search" />
+        </Button>
+    </Tooltip>
+    <Tooltip
+        actionHeader="Search shortcuts"
+        keyCombinationsActions={[
+            { eventKeyCombination: ["Command", "K"] },
+        ]}
+    >
+        <Button aria-keyshortcuts="command+k" variant="primary">
+            <SearchIcon title="search" />
+        </Button>
+    </Tooltip>
+</Stack>`.trim();
+
+export const useKeyCombinationExample = `const HookExample = () => {
+    const [combinationTriggeredText, setCombinationTriggeredText] = React.useState("");
+    useKeyCombination({
+        keys: ["Shift", "q"],
+        onCombinationPress: () => {
+            setCombinationTriggeredText("Shift + Q pressed");
+        },
+    });
+
+    return (
+        <>
+            <Paragraph>Press the <KeyboardKeyGroup><KeyboardKey>Shift</KeyboardKey><KeyboardKey>Q</KeyboardKey></KeyboardKeyGroup> key to reveal the pressed states below:</Paragraph>
+            <Paragraph>Combination triggered: {combinationTriggeredText}</aragraph>
+        </>
+    )
+}
+
+render(
+    <HookExample />
+)`.trim();
+
+export const useKeyCombinationsExample = `const HookExample = () => {
```

Comment:
issue: this and the above example don't seem to be fully working/interactive

Reply:
Could you test them again? I'm not having any issue my end. Make sure you have clicked inside the window so it's active.

Reply:
working now! it was feeling a little finicky but I can't put my finger on it. I think it's all good

Reply:
😟 Let's keep testing as a team but I feel I got all the issue worked out.